Reflections/Wk1.txt

```diff
@@ -0,0 +1,5 @@
+1. This week I have finished up research with my APIs and have chosen the Yelp API. After that I watched some Youtube Videos and discovered a block code website much like AppInventor. After that I was able to create a basic prototype to show off in class. I created two pages, the menu page with a distance slider and the results page which is abloe to display the name of the restaurant, the phone number,  a picture of the restaurant, a reroll button and a more info button.
+2. The biggest challenge I faced was learning the new website. Although it is very similar to AppInventor, it still has new  features that I wasn't sure how worked. I used a lot of the websites tutorials to figue out which each thing did.
+3. In the following week I plan to rework the design of the app to move the distance slider to a different page of the app and then add more filters (type and price) for the restaurant selcetion. I need to do some research about implementing the wheel. I already have the picture created for the wheel just need it to spin. Finally I want to add a get directions button rather than the user grabbing the location from the yelp website.

```

Comment:
These three goals have some components of S.M.A.R.T. goals, however some of the components are missing. For the next reflection, do not forget to add the goals with an estimated number of hours/day to complete each goal and to also work on having the five components of S.M.A.R.T. goals: Specific, Measurable, Attainable, Relevant, and Time-Bound.